components/board/genericlinux/gpio.go

```diff
@@ -36,6 +36,9 @@ type gpioPin struct {
 	mu        sync.Mutex
 	cancelCtx context.Context
 	logger    logging.Logger
+
+	swPWMContext context.Context
+	swPWMCancel  func()
```

Comment:
I'd prefer to have lowercase W and M in these names, so it's easier to chunk things together into words as you read it. In this context, "PWM" is a complete word. (I recognize that our codebase is inconsistent on this front, and this is a personal opinion. If one of our style guides talks about this explicitly, we should do whatever it says. but I haven't seen a policy on this myself).